PxWeb.UnitTests/Search/LuceneAnalyzerTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Px.Search;
+using Px.Search.Lucene;
+using Px.Search.Lucene.Config;
+
+namespace PxWeb.UnitTests.Search
+{
+    [Ignore("These tests require that you index your databases first. And perhaps they test Lucene as much as local code :-)")]
```

Comment:
Can't we just commit the _INDEX folder? It was done for PxWeb :-)

Reply:
@likp @runejo If the repo to should be "clone, build and run" we will need this(_INDEX) for seach to work, wont we. Let's talk about it on teams tomorrow.

Reply:
We decided to add the index.